innopoints/views/project.py

```diff
@@ -99,6 +91,61 @@ def list_projects():
     return schema.jsonify(db_query.all())
 
 
+@api.route('/projects/past')
+def list_past_projects():
+    """List past projects."""
+    default_page = 1
+    default_limit = 12
+
+    db_query = Project.query.filter(or_(Project.lifetime_stage == LifetimeStage.finalizing,
+                                        Project.lifetime_stage == LifetimeStage.finished))
+    count_query = db.session.query(db.func.count(Project.id)).filter(
+        or_(Project.lifetime_stage == LifetimeStage.finalizing,
+            Project.lifetime_stage == LifetimeStage.finished)
+    )
+    if 'q' in request.args:
+        like_query = f'%{request.args["q"]}%'
+        db_query = db_query.join(Project.activities).filter(
+            or_(Project.title.ilike(like_query),
+                Activity.name.ilike(like_query),
+                Activity.description.ilike(like_query))
+        ).distinct()
+        count_query = count_query.join(Project.activities).filter(
+            or_(Project.title.ilike(like_query),
+                Activity.name.ilike(like_query),
+                Activity.description.ilike(like_query))
+        ).distinct()
+
+    try:
+        limit = int(request.args.get('limit', default_limit))
+        page = int(request.args.get('page', default_page))
+    except ValueError:
+        abort(400, {'message': 'Bad query parameters.'})
+
+    if limit < 1 or page < 1:
+        abort(400, {'message': 'Limit and page number must be positive.'})
+
+    db_query = db_query.order_by(Project.creation_time.desc())
+    db_query = db_query.offset(limit * (page - 1)).limit(limit)
+
+    conditional_exclude = ['review_status', 'moderators']
+    if current_user.is_authenticated:
+        conditional_exclude.remove('moderators')
+        if not current_user.is_admin:
+            conditional_exclude.remove('review_status')
+    exclude = ['admin_feedback', 'review_status', 'files', 'image_id',
```

Comment:
`review_status` is in both `exclude` and `conditional_exclude`

Reply:
also, why exclude the image?

Reply:
The `image_id` is excluded in favor of the `image_url` property